src/components/App/AppIFrame.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import { styled } from '@aragon/ui'
+import { clamp, lerp } from '../../math-utils'
+import { noop } from '../../utils'
+import AppLoadingProgressBar from './AppLoadingProgressBar'
+
+const LOADING_START = 25 // Start loading indicator at 25%
+const LOADING_END = 100
+const LOADING_FUDGE_LIMIT = 75 // Limit arbitrary incremental movement of loading incator to 75%
+
+// See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe for details about sandbox
+// `sandbox` works like a whitelist: by default, almost every functionality is restricted.
+const SANDBOX = [
+  // Allows new browsing contexts (e.g. new windows or tabs) generated by the
+  // iframe to escape the sandbox restrictions. Otherwise, new contexts would
```

Comment:
👍 can we do some tests to make sure that's the case and it is impossible for it to run any JS in the parent

Reply:
I tested it spuriously while playing around with iframes earlier, but thinking about it, we could put up some random tests under a repo in aragonlabs.

It should be noted that the demo app isn't able to load any assets (e.g. fonts; you'll see a bunch of errors in the console) from the same origin because of the origin sandbox that's been set 😅.

Reply:
In the future this should be part this project's test suite, but I imagine we would need to run it against a headless browser or something. Nothing too urgent, but this could be an attack vector.

Reply:
What @onbjerg says in the comment below could be a cool test case, make a super malicious Aragon app that tries to break the sandbox and do nasty stuff ;)

Reply:
Added https://github.com/aragon/aragon/issues/87.